app/src/main/java/com/theoplayer/android/connector/MainActivity.kt

```diff
@@ -45,15 +45,20 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun setupConviva() {
-        val customerKey = "your_conviva_customer_key"
-        val gatewayUrl = "your_conviva_debug_gateway_url"
+        val customerKey = "876a2328cc34e791190d855daf389567c96d1e86"
```

Comment:
Is it a problem that we are exposing some customer key?

Reply:
It's our test account, and it's being shared in demo code for connnectors on other platforms as well, so not a real issue  I think. But I removed it just to be sure :)